src/main/cpp/windows/jssc.cpp

```diff
@@ -29,7 +29,16 @@
 #include <jssc_SerialNativeInterface.h>
 #include "version.h"
 
-//#include <iostream>
+#if defined(_MSC_VER) && _MSC_VER < 1800
```

Comment:
This code is redundant and slightly anti-pattern since it contains conditions which should never exist (e.g. checking for `_MSC_VER < 1800` on Unix).  Would it instead make sense to move this to a common header file?  What's the normal course of action for helpers like this?